onnxruntime/test/providers/cpu/signal/signal_ops_test.cc

```diff
@@ -115,11 +145,16 @@ static void TestDFTInvertible(bool complex) {
   for (int64_t axis = 1; axis < 2; axis += 1) {
     for (int64_t signal_dim1 = 2; signal_dim1 <= 5; signal_dim1 += 1) {
       for (int64_t signal_dim2 = 2; signal_dim2 <= 5; signal_dim2 += 1) {
-        DFTInvertibleTester test(axis);
+        DFTInvertibleTester test(axis, since_version);
         vector<int64_t> input_shape{num_batches, signal_dim1, signal_dim2, 1 + (complex ? 1 : 0)};
         vector<float> input_data = random.Uniform<float>(input_shape, -100.f, 100.f);
         test.AddInput("input", input_shape, input_data);
 
+        if (since_version >= kOpsetVersion20) {
```

Comment:
Do you have a test covering opset > 20 and no specified axis?

Reply:
kOpsetVersion20 is used as since_version so the test will cover opset > 20. Tests to cover new or changed features of future opset shall be added in the future.
TestNaiveDFTFloat etc. are test w/o axis specified.